docs/machine-output.md

```diff
@@ -0,0 +1,125 @@
+# Machine Readable Output
+
+This document outlines all the machine readable output options and examples.
+
+`$ odo url create -o json | jq`
```

Comment:
I think we should mention before that the `jq` tool is required for pretty formatting

Reply:
+1

Reply:
Done :+1: